dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature

```diff
@@ -21,6 +21,6 @@ Subsystem-Name: Jakarta XML Bindings 3.0
   com.ibm.websphere.appserver.eeCompatible-9.0, \
   io.openliberty.jakarta.jaxb-3.0
 -bundles=\
-  io.openliberty.jaxb.3.0.internal.tools
+  io.openliberty.jaxb.internal.tools.3.0
```

Comment:
I think this was named correctly, why change it?

Reply:
@tevans78 so I may be misremembering, but I thought that the ResourceProvider had to match the bundle name exactly (with the exception of using under scores for the versioning). As a part of getting the feature working correctly, I changed the bundle name to match the ResourceProvider.

This is what it is presently:
```
Service-Component: \
  io.openliberty.jaxb.internal.tools_3_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
```
I attempted this locally, and the Jakarta ContextFactory Service still wasn't getting picked up:
```
Service-Component: \
  io.openliberty.jaxb_3_0_internal.tools_${replace;${bVersion};\\.;_}.ResourceProvider; \
```